zio-http/src/main/scala/zhttp/experiment/Router.scala

```diff
@@ -34,7 +36,7 @@ object Router {
     def /[B, C](other: Router[B])(implicit ev: Combine.Aux[Unit, B, C]): Router[C] = ???
   }
 
-  val route = Method.GET / "a" / "b" / ![Int] / "c" / ![String] / ![Int]
+  // val route = Method.GET / "a" / "b" / ![Int] / "c" / ![String] / ![Int]
```

Comment:
```suggestion
  val route = Method.GET / "a" / "b" / ![Int] / "c" / ![String] / ![Int]
```

Reply:
In Scala 3 ! operator is not working renamed it to Arg, Also Combine is not working for 3 variables